### ISSUE TITLE
Unhandled Exception by call gui and monitor

### ISSUE THREAD

**Issue by mhbosch** [microsoft/AttackSurfaceAnalyzer#565]
**Describe the bug**

**To Reproduce**
Call GUI 

```
Content root path: C:\Users\posts\Downloads\ASA_win_2.2.75
Now listening on: http://127.0.0.1:5000
Application started. Press Ctrl+C to shut down.
Unhandled exception. Unhandled exception. Unhandled exception. Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.Data.Sqlite.SqliteConnection.RemoveCommand(SqliteCommand command)
   at AttackSurfaceAnalyzer.Utils.DatabaseManager.WriteFileMonitor(FileMonitorObject fmo, String RunId) in D:\a\1\s\Lib\Utils\DatabaseManager.cs:line 979
   at AttackSurfaceAnalyzer.Collectors.FileSystemMonitor.WriteChange(FileSystemEventArgs objIn, NotifyFilters filters) in D:\a\1\s\Lib\Collectors\FileSystemMonitor.cs:line 223
   at System.IO.FileSystemWatcher.NotifyFileSystemEventArgs(WatcherChangeTypes changeType, ReadOnlySpan`1 name)
   at System.IO.FileSystemWatcher.ParseEventBufferAndNotifyForEach(Byte[] buffer, UInt32 numBytes)
   at System.IO.FileSystemWatcher.ReadDirectoryChangesCallback(UInt32 errorCode, UInt32 numBytes, AsyncReadState state)
   at System.IO.FileSystemWatcher.<>c.<StartRaisingEvents>b__83_0(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* overlappedPointer)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pNativeOverlapped)
System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.Data.Sqlite.SqliteConnection.RemoveCommand(SqliteCommand command)
   at AttackSurfaceAnalyzer.Utils.DatabaseManager.WriteFileMonitor(FileMonitorObject fmo, String RunId) in D:\a\1\s\Lib\Utils\DatabaseManager.cs:line 979
   at AttackSurfaceAnalyzer.Collectors.FileSystemMonitor.WriteChange(FileSystemEventArgs objIn, NotifyFilters filters) in D:\a\1\s\Lib\Collectors\FileSystemMonitor.cs:line 223
   at System.IO.FileSystemWatcher.NotifyFileSystemEventArgs(WatcherChangeTypes changeType, ReadOnlySpan`1 name)
   at System.IO.FileSystemWatcher.ParseEventBufferAndNotifyForEach(Byte[] buffer, UInt32 numBytes)
   at System.IO.FileSystemWatcher.ReadDirectoryChangesCallback(UInt32 errorCode, UInt32 numBytes, AsyncReadState state)
   at System.IO.FileSystemWatcher.<>c.<StartRaisingEvents>b__83_0(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* overlappedPointer)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pNativeOverlapped)
System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.Data.Sqlite.SqliteConnection.RemoveCommand(SqliteCommand command)
   at AttackSurfaceAnalyzer.Utils.DatabaseManager.WriteFileMonitor(FileMonitorObject fmo, String RunId) in D:\a\1\s\Lib\Utils\DatabaseManager.cs:line 979
   at AttackSurfaceAnalyzer.Collectors.FileSystemMonitor.WriteChange(FileSystemEventArgs objIn, NotifyFilters filters) in D:\a\1\s\Lib\Collectors\FileSystemMonitor.cs:line 223
   at System.IO.FileSystemWatcher.NotifyFileSystemEventArgs(WatcherChangeTypes changeType, ReadOnlySpan`1 name)
   at System.IO.FileSystemWatcher.ParseEventBufferAndNotifyForEach(Byte[] buffer, UInt32 numBytes)
   at System.IO.FileSystemWatcher.ReadDirectoryChangesCallback(UInt32 errorCode, UInt32 numBytes, AsyncReadState state)
   at System.IO.FileSystemWatcher.<>c.<StartRaisingEvents>b__83_0(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* overlappedPointer)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pNativeOverlapped)
System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.Data.Sqlite.SqliteConnection.RemoveCommand(SqliteCommand command)
   at AttackSurfaceAnalyzer.Utils.DatabaseManager.WriteFileMonitor(FileMonitorObject fmo, String RunId) in D:\a\1\s\Lib\Utils\DatabaseManager.cs:line 979
   at AttackSurfaceAnalyzer.Collectors.FileSystemMonitor.WriteChange(FileSystemEventArgs objIn, NotifyFilters filters) in D:\a\1\s\Lib\Collectors\FileSystemMonitor.cs:line 223
   at System.IO.FileSystemWatcher.NotifyFileSystemEventArgs(WatcherChangeTypes changeType, ReadOnlySpan`1 name)
   at System.IO.FileSystemWatcher.ParseEventBufferAndNotifyForEach(Byte[] buffer, UInt32 numBytes)
   at System.IO.FileSystemWatcher.ReadDirectoryChangesCallback(UInt32 errorCode, UInt32 numBytes, AsyncReadState state)
   at System.IO.FileSystemWatcher.<>c.<StartRaisingEvents>b__83_0(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* overlappedPointer)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pNativeOverlapped)
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.Data.Sqlite.SqliteConnection.RemoveCommand(SqliteCommand command)
   at AttackSurfaceAnalyzer.Utils.DatabaseManager.WriteFileMonitor(FileMonitorObject fmo, String RunId) in D:\a\1\s\Lib\Utils\DatabaseManager.cs:line 979
   at AttackSurfaceAnalyzer.Collectors.FileSystemMonitor.WriteChange(FileSystemEventArgs objIn, NotifyFilters filters) in D:\a\1\s\Lib\Collectors\FileSystemMonitor.cs:line 223
   at System.IO.FileSystemWatcher.NotifyFileSystemEventArgs(WatcherChangeTypes changeType, ReadOnlySpan`1 name)
   at System.IO.FileSystemWatcher.ParseEventBufferAndNotifyForEach(Byte[] buffer, UInt32 numBytes)
   at System.IO.FileSystemWatcher.ReadDirectoryChangesCallback(UInt32 errorCode, UInt32 numBytes, AsyncReadState state)
   at System.IO.FileSystemWatcher.<>c.<StartRaisingEvents>b__83_0(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* overlappedPointer)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pNativeOverlapped)
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.Data.Sqlite.SqliteConnection.RemoveCommand(SqliteCommand command)
   at AttackSurfaceAnalyzer.Utils.DatabaseManager.WriteFileMonitor(FileMonitorObject fmo, String RunId) in D:\a\1\s\Lib\Utils\DatabaseManager.cs:line 979
   at AttackSurfaceAnalyzer.Collectors.FileSystemMonitor.WriteChange(FileSystemEventArgs objIn, NotifyFilters filters) in D:\a\1\s\Lib\Collectors\FileSystemMonitor.cs:line 223
   at System.IO.FileSystemWatcher.NotifyFileSystemEventArgs(WatcherChangeTypes changeType, ReadOnlySpan`1 name)
   at System.IO.FileSystemWatcher.ParseEventBufferAndNotifyForEach(Byte[] buffer, UInt32 numBytes)
   at System.IO.FileSystemWatcher.ReadDirectoryChangesCallback(UInt32 errorCode, UInt32 numBytes, AsyncReadState state)
   at System.IO.FileSystemWatcher.<>c.<StartRaisingEvents>b__83_0(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* overlappedPointer)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pNativeOverlapped)
```

The Folder D:\a\1\s\Lib\Collectors\ exit not.

Edition	Windows 10 Pro
Version	20H2
Betriebssystembuild	19042.685
Leistung	Windows Feature Experience Pack 120.2212.551.0

**Comment by gfs** [comment]
Thanks for your report.  I have reproduced this and narrowed it down to windows only.

**Comment by gfs** [comment]
Fixed by #566.  New version should publish in about 30 minutes.
